spec/index.md

```diff
@@ -14,6 +14,15 @@ lang: en
 
 This is version 1.0.0 of the specification.
 
+- [Terminology](#terminology)
```

Comment:
Does the ToC need a heading? Otherwise it looks like it's part of the "version" section.
We could also move the version to be placed after the ToC.

Reply:
Done in https://github.com/microbit-foundation/spec-power-management/pull/4/commits/536de62afe0990a39ee90e5f8ee963f389883415